src/components/tl/ucp/reduce_scatter/reduce_scatter_ring.c

```diff
@@ -112,7 +114,10 @@ static void ucc_tl_ucp_reduce_scatter_ring_progress(ucc_coll_task_t *coll_task)
 
     sendto   = ucc_ep_map_eval(task->reduce_scatter_ring.inv_map, sendto);
     recvfrom = ucc_ep_map_eval(task->reduce_scatter_ring.inv_map, recvfrom);
-
+    if (team->cfg.use_reordering) {
```

Comment:
should we put if/else branching here ?

Reply:
what do you mean? what should be in else?

Reply:
nevermind